script/specific/DeployMKR-Feb26ProposalPayload.s.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@forge-std/console.sol";
+import {Script} from "@forge-std/Script.sol";
+import {ProposalPayload} from "../../src/payloads/MKRPayload-Feb26.sol";
+
+contract DeployProposalPayload is Script {
+    function run() external {
+        vm.startBroadcast();
+        ProposalPayload proposalPayload = new ProposalPayload();
+        console.log("Proposal Payload address", address(proposalPayload));
```

Comment:
No need to console, broadcast action will log all needed info

Reply:
done

Reply:
I still see the console here. ?

Reply:
no more :)